doc/host-setup.md

```diff
@@ -146,3 +146,37 @@ Checking the current value again will reveal that the helper is now disabled
 # cat /sys/module/drbd/parameters/usermode_helper
 disabled
 ```
+
+## Disable Lvmetad on CentOS 7 and Ubuntu 18.04
+
+Reference: https://access.redhat.com/solutions/2053483
+
+CentOS 7 and Ubuntu 18.04 by default use `lvmetad`,  a metadata caching daemon for LVM. The daemon improves performance of LVM commands by avoiding rescanning the disks on every execution. `piraeus-ns-node` pods run LVM commands inside a container without access to lvmetad. Since that means  `lvmetad` would get out of sync, it is recommended to disable `lvmetad` in CentOS 7 and Ubuntu 18.04 Bionic.
+
+Note: Newer distributions no longer include with `lvmetad`, no changes necessary.
+
+Follow below steps to disable lvmetad completely:
+```bash
+$ systemctl stop lvm2-lvmetad.socket lvm2-lvmetad.service
+
+$ systemctl disable lvm2-lvmetad.service
+
+$ systemctl mask lvm2-lvmetad.socket
+
+$ sed -i 's/use_lvmetad = 1/use_lvmetad = 0/' /etc/lvm/lvm.conf
+
+$ cat /etc/lvm/lvm.conf | grep use_lvmetad
+use_lvmetad = 0
+```
+If the root partition already uses LVM, you also need to update the initial ram-disk:
+```bash
+# CentOS 7
+$ cp -vf /boot/initramfs-$(uname -r).img /boot/initramfs-$(uname -r).img.$(date +%m-%d-%H%M%S).bak
+$ dracut -f -v
+
+# Ubuntu 18.04
+$ cp -vf /boot/initrd.img-$(uname -r) /boot/initd.img.$(uname -r).$(date +%m-%d-%H%M%S).bak
+$ update-initramfs -c -k $(uname -r)
+$ update-grub
+```
+
```

Comment:
Weird that this wasn't caught by your `pre-commit` run, but this still fails here. Just remove this line, I think thats the last blocker

Reply:
As per vscode markdownlint suggestion, I added blank lines before and after code block. `git commit` passed with no error per `pre-commit`: 

```
 git commit -s -m 'clear syntax errors posted by markdownlint'
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check yaml...........................................(no files to check)Skipped
check for added large files..............................................Passed
gofumpt..............................................(no files to check)Skipped
golangci-lint........................................(no files to check)Skipped
go-unit-tests........................................(no files to check)Skipped
generated deep-copy code must be up-to-date..........(no files to check)Skipped
generated crds must be up-to-date....................(no files to check)Skipped
generate helm values for CN users....................(no files to check)Skipped
generate full yaml deployment from helm templates....(no files to check)Skipped
[master be83e4e] clear syntax errors posted by markdownlint
 1 file changed, 4 insertions(+), 2 deletions(-)
```
Please see if it can be merged.